agent/05_agent_configure.sh

```diff
@@ -522,3 +589,5 @@ fi
 generate_cluster_manifests
 
 generate_extra_cluster_manifests
+
+write_extra_workers_ips
```

Comment:
It is exported but only available within 05_agent_configure.sh and its child processes. 07_agent_add_node.sh does not see it.

Reply:
Ok, to avoid creating an unnecessary artifact on the disk the suggestion would be to retrieve such information from the `ostestbm` libvirt network - in case the exported vars would result empty. In that way also the scenario where the step 07 is executed in standalone _after_ having completed the previous (deploy) workflow (which is something definitely useful while developing) should work. Fine for me to address this change in subsequent PR